RecoTracker/TkSeedGenerator/plugins/DeepCoreSeedGenerator.h

```diff
@@ -0,0 +1,177 @@
+#ifndef RecoTracker_TkSeedGenerator_DeepCoreSeedGenerator_H
+#define RecoTracker_TkSeedGenerator_DeepCoreSeedGenerator_H
+
+#define jetDimX 30  //pixel dimension of NN window on layer2
+#define jetDimY 30  //pixel dimension of NN window on layer2
+#define Nlayer 4    //Number of layer used in DeepCore
+#define Nover 3     //Max number of tracks recorded per pixel
+#define Npar 5      //Number of track parameter
```

Comment:
common constants should be moved to a common place and defined as normal C++ named values, not a preprocessor macros (definitely not as macros in header files)

Reply:
ok, fixed in the next commit